### DISCUSSION TITLE
Training times increased 10x with a second video

### DISCUSSION THREAD

**Discussion by tomhagley6** [talmolab/sleap#1158] [Help!]
For initial training (one 5-min video with 25 labelled frames), my training epochs took around 40 seconds. Now I've added a second video to my project, predicted every frame in a second 5-min video, labelled 40 of them, and then re-run training on this new labelled project. My training epochs are now taking around 600 seconds to complete.

Any ideas why this might be? Something to do with having predicted every frame of the second video or just having multiple videos in a project maybe?

Thanks!

**Comment by roomrys** [comment]
HI @tomhagley6,

Sorry for the delay! Are you able to share the training_config.json and slp file with me at [lmaree@salk.edu](mailto:lmaree@salk.edu)?

Thanks,
Liesl
